main.js

```diff
@@ -9,6 +9,8 @@ const autoUpdaterInterval = 60 * 60 * 1000;
 const ipc = electron.ipcMain;
 const Menu = electron.Menu;
 const shell = electron.shell;
+const ElectronConfig = require('electron-config');
+const userConfig = new ElectronConfig();
```

Comment:
That `new` statement does pull the current `appPath` directory, so I moved it lower in the file.